nvflare/lighter/impl/static_file.py

```diff
@@ -104,6 +113,25 @@ def _build_overseer(self, overseer: Participant, ctx: ProvisionContext):
         else:
             ctx[PropKey.OVERSEER_END_POINT] = f"{protocol}://{overseer.name}{api_root}"
 
+    @staticmethod
+    def _build_conn_properties(site: Participant, ctx: ProvisionContext, site_config: dict):
+        valid_values = [ConnSecurity.CLEAR, ConnSecurity.INSECURE, ConnSecurity.TLS, ConnSecurity.MTLS]
```

Comment:
what's the difference between CLEAR vs INSECURE?

Reply:
The same. Just that "clear" is more of a word that users may prefer over "insecure". Remember that these are the words that the user will enter into the yml file.